sql-cli/sql_cli/dag_generator.py

```diff
@@ -122,4 +123,11 @@ def generate_dag(directory: Path, dags_directory: Path, generate_tasks: bool) ->
         context={"dag": workflow_files_dag},
         output_file=output_file,
     )
+
+    # format DAG to be black compliant
+    with open(output_file, "r+") as file:
+        output_str = format_str(file.read(), mode=FileMode())
```

Comment:
Please use pathlib's [read_text](https://docs.python.org/3/library/pathlib.html#pathlib.Path.read_text)

Could we create a small helper func for this, please? :)